applications/FluidDynamicsApplication/python_scripts/navier_stokes_ale_fluid_solver.py

```diff
@@ -96,17 +96,17 @@ def _ManipulateMeshMotionSolverSettingsForMeshVelocityComputation(cls, fluid_sol
                     info_msg  = '"time_scheme" of the fluid (bdf2) '
                     info_msg += 'is different from the\n"time_scheme" used for the '
                     info_msg += 'calculation of the mesh-velocity (' + time_scheme_mesh_vel + ')'
-                    KM.Logger.PrintInfo("::[NavierStokesAleFluidSolver]::", info_msg)
+                    KM.Logger.PrintInfo(self.__class__.__name__, info_msg)
```

Comment:
this could be an alternative:

https://github.com/KratosMultiphysics/Kratos/blob/3357747801b79feea639fe2f89ad9076f44a34ff/applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_predictor.py#L59-L61

it only does have to be defined once in the baseclass, but still prints the correct names

Reply:
Maybe we should add this to the base Python solver too.